README.md

```diff
@@ -13,30 +13,41 @@ pip3 install -r requirements.txt
 The tests suite requires a Substra network up and running. The network can be started
 either with skaffold (Kubernetes), with docker-compose, or manually.
 
+Make sure you have pulled the different substra-tools images:
+```
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.0.1
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.1.0
+$> docker pull eu.gcr.io/substra-208412/substra-tools:0.2.0
```

Comment:
As `substra-tools` is now open source, we should use `docker pull substrafoundation/substra-tools:0.0.1` only no ?

Reply:
It may have side effect, as we will need to update some files like Dockerfile for instance.